lightning_uq_box/uq_methods/laplace_model.py

```diff
@@ -28,29 +27,44 @@
 # over training_step in lightning
 
 
-def tune_prior_precision(
-    model: Laplace, tune_precision_lr: float, n_epochs_tune_precision: int
+def tune_prior_precision_and_sigma(
+    model: Laplace,
+    tune_precision_lr: float,
+    n_epochs_tune_precision: int,
+    tune_prior_precision: bool,
+    tune_sigma_noise: bool,
```

Comment:
The function is called using those in `on_test_start()` but they are not used in the function, as far as I can see.

Reply:
sorry, forgot to push again, this was an earlier version. Thanks for the catch!